service-discovery/stork-custom/src/main/java/io/quarkus/ts/stork/custom/SimpleLoadBalancerProvider.java

```diff
@@ -0,0 +1,17 @@
+package io.quarkus.ts.stork.custom;
+
+import io.smallrye.stork.api.LoadBalancer;
+import io.smallrye.stork.api.ServiceDiscovery;
+import io.smallrye.stork.api.config.LoadBalancerAttribute;
+import io.smallrye.stork.api.config.LoadBalancerType;
+import io.smallrye.stork.spi.LoadBalancerProvider;
+
+@LoadBalancerType("simple")
+@LoadBalancerAttribute(name = "my-attribute", description = "Attribute that alters the behavior of the LoadBalancer", required = false)
```

Comment:
It seems to me that this attribute is never used. It it still usefull here? It influences the contents of the respective `Configuration`, but we do not verify that anywhere. So I suggest to remove it.